src/object_list.c

```diff
@@ -29,6 +29,9 @@
 #include "world/scenery.h"
 #include "world/water.h"
 
+// for qsort()
+#include <stdlib.h>
```

Comment:
I was initially testing with Linux's `qsort_r` and compiler complained about the function being undefined. Updated now.